packages/lodestar-cli/src/network/fileEnr.ts

```diff
@@ -0,0 +1,46 @@
+import {createKeypairFromPeerId, ENR, ENRKey, ENRValue} from "@chainsafe/discv5";
+import {writeFileSync, readFileSync} from "fs";
+import PeerId from "peer-id";
+
+/**
+ * `FileENR` is an `ENR` that saves the ENR contents to a file on every modification
+ */
+export class FileENR extends ENR {
+  private filename: string;
+  private localPeerId: PeerId;
+
+  constructor(filename: string, peerId: PeerId) {
+    super();
+    this.filename = filename;
+    this.localPeerId = peerId;
+  }
+
+  static initFromFile(filename: string, peerId: PeerId): FileENR {
+    const enr = FileENR.decodeTxt(readFileSync(filename, "utf8")) as FileENR;
+    return this.initFromENR(filename, peerId, enr);
+  }
+
+  static initFromENR(filename: string, peerId: PeerId, enr: FileENR): FileENR {
```

Comment:
Add a unit test for `initFromENR` to make sure the prototype manipulation is ok

Reply:
`initFromFile` calls `initFromENR`, so the logic is tested there.